app/views/RoomInfoView/components/RoomInfoButtons.tsx

```diff
@@ -93,15 +102,16 @@ export const RoomInfoButtons = ({
 				label={i18n.t(isIgnored ? 'Unignore' : 'Ignore')}
 				iconName='ignore'
 				showIcon={!!renderIgnoreUser}
-				danger
+				danger={!renderReportUser}
 			/>
 			<BaseButton
 				onPress={handleBlockUser}
-				label={i18n.t(`${isBlocked ? 'Unblock' : 'Block'}_user`)}
+				label={i18n.t(`${isBlocked ? 'Unblock' : 'Block'}`)}
```

Comment:
You need to remove those i18n keys ending with `_user`.

Reply:
Actually, we already use the Block_user and Unblock_user in RoomActionsView -> https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/e295cbbcc65ad3b8953152dcaedc0d8ee002b8e4/app/views/RoomActionsView/index.tsx#L869

Reply:
I'm talking about unifying behaviors.
Unless we're intentionally adding new keys for similar features, we should stick to one of them.

Reply:
Ok, I talked with Jose and we can unify both as `Block`